### ISSUE TITLE
Auto-formatting

### ISSUE THREAD

**Issue by Kyslik** [DavidAnson/vscode-markdownlint#13]
Is there a way how to apply lint rules via shortcut?

**Comment by DavidAnson** [comment]
When installed, `vscode-markdownlint` runs its rules after every change to the document. What would the shortcut do differently? Can you give more detail, please?

**Comment by Kyslik** [comment]
What I mean is actually auto-format it.

So lets say I have a document readme.md like

```markdown
## About
Some information about...
```
So linter yells at me with rule 2, 22 and 41 I want to hit a shortcut to automatically apply suggestions, without actually manually amending the file.

Is that possible?

**Comment by DavidAnson** [comment]
Got it! In some cases, I think it should be possible to safely modify the file to fix issues. I'll think more about this. Thanks!

**Comment by Kyslik** [comment]
I think it should be separate package since this is "linter" and should only lint. Just my 2 cents. I have some free time, do you have any source (tutorials, guides) on how to create ext for VSCode?

**Comment by Danilux** [comment]
I was looking for the same feature, eslint has "eslint.autoFixOnSave": true, and it's so sweet, please consider this.

**Comment by Kyslik** [comment]
Thank you :) I will check it out.

**Comment by ChrisChinchilla** [comment]
@DavidAnson Following up here, from my experiences with Atom, tidy markdown seems to format code based on the same rules that mdlint follows, so in theory, this extension should work

https://github.com/Tehnix/vscode-tidymarkdown

**Comment by Ilias48rus** [comment]
@Kyslik , For you to note, "Auto" mean **doesn't require a trigger**(which is the "shortcut" in your case).

**Comment by Kyslik** [comment]
@Ilias48rus renamed
